app/src/main/java/com/example/concordia_campus_guide/Activities/RoutesActivity.java

```diff
@@ -181,11 +184,14 @@ private void setRoutesAdapter() {
         allRoutes.setOnItemClickListener(new AdapterView.OnItemClickListener() {
             @Override
             public void onItemClick(AdapterView<?> adapterView, View view, int i, long l) {
-                Intent openPaths = new Intent(RoutesActivity.this,
```

Comment:
You are disabling the option to go to the paths view for all, not only the shuttle